java-checks-test-sources/default/src/main/java/checks/StringLiteralDuplicatedCheckSample.java

```diff
@@ -113,3 +113,65 @@ class Coverage {
   private final String prevLeftNull = "SELECT" + 3;
   private final String prevRightNull = 3 + "SELECT";
 }
+
+class ExceptionArguments {
+  private void simple(int r) {
```

Comment:
Let's help our future selves by renaming some of this methods or adding some comments.
```suggestion
class DuplicatedExceptionArguments {
  private void areCompliantByDefault(int r) {
```